draft-ietf-jsonpath-base.md

```diff
@@ -534,8 +534,7 @@ member value of the input
 node corresponding to the member name `"a"`.
 The result is again a list of one node: `[{"b":0},{"b":1},{"c":2}]`.
 
-Next, `[*]` selects from any input node which is an array and selects all the elements
-of the input node.
+Next, `[*]` selects from any input node — either an array or an object — all its elements or members.
```

Comment:
You can't select members, only member values.

Reply:
I overlooked, that Glyn's original text was correct, as he was referring to that specific example. So possibly better
```
Next, `[*]` selects from any input node — an array here — all its elements.
``` 
@cabo is right regarding members though.